python_modules/dagster/dagster/_core/definitions/automation_tick_evaluation_context.py

```diff
@@ -165,6 +167,7 @@ def _get_updated_evaluations(
                 updated_evaluations.append(result.serializable_evaluation)
         return updated_evaluations
 
+    @use_partition_loading_context
     async def async_evaluate(
```

Comment:
i'm slightly surprised that this is the only callsite that needed a new parittion loading context

Do we have coverage on the graphql read layer for evaluations involving dynamic partition subsets? Like places where we display the set of partitionts that were considered?

Reply:
rather than doing this piecemeal, I stacked something at the bottom of the stack to just make BaseWorkspaceRequestContext a contextmanager that we enter at the beginning of a graphql query. this means that every single gql query will have a partition loading context available

verified manually that I hit gql errors before that change and did not hit them after